addon_service/addon_operation_invocation/models.py

```diff
@@ -84,10 +84,11 @@ def clean_fields(self, *args, **kwargs):
                 {"thru_addon": "thru_addon and thru_account must agree"}
             )
 
+    @property
```

Comment:
nit: if this is being turned into a property, please relocate it with the other properties